system/admin/filesystem.go

```diff
@@ -1,10 +1,38 @@
 package admin
 
 import (
+	"encoding/json"
 	"net/http"
 	"os"
+	"strings"
+
+	"github.com/ponzu-cms/ponzu/system/db"
+	"github.com/ponzu-cms/ponzu/system/item"
 )
 
+func deleteUploadFromDisk(target string) error {
+	// get data on file
+	data, err := db.Upload(target)
+	if err != nil {
+		return err
+	}
+
+	// unmarshal data
+	upload := item.FileUpload{}
+	if err = json.Unmarshal(data, &upload); err != nil {
+		return err
+	}
+
+	// use path to delete the physical file from disk
+	delPath := strings.Replace(upload.Path, "/api/", "./", 1)
```

Comment:
at quick glance, does this keep the unix-y filepaths? might need to split this on `/` and filepath.Join the results. sorry if this is just me missing while on mobile-only

Reply:
Unsure. I've done nothing to make the ```/``` cross compile compatible. Let me know.